x/incentives/keeper/distribution.go

```diff
@@ -180,7 +182,10 @@ func (k Keeper) rewardParticipants(
 				// Cap rewards in mint denom (i.e. aevmos) to receive only up to 100% of
 				// the participant's gas spent and prevent gaming
 				if mintDenom == allocation.Denom {
-					reward = sdk.MinDec(reward, cumulativeGas)
+					params := k.GetParams(ctx)
+					rewardScaler := params.RewardScaler
```

Comment:
let's move this below the `mintDenom` var so that we don't have to call it on every iteration 👍